packages/nc-gui/components/erd/SimpleView.vue

```diff
@@ -0,0 +1,227 @@
+<script setup lang="ts">
```

Comment:
Use a meaningful component name.
`SimpleView` says nothing about what this does.
Maybe use `Flow` as a name to suggest that this component renders the actual Flowchart for the ERD?